content/en/getting_started/agent/autodiscovery.md

```diff
@@ -76,76 +76,18 @@ Setting up Autodiscovery for your infrastructure requires the following two step
 
 ### Enable Autodiscovery
 
-#### With the Agent on a host
-
-{{< tabs >}}
-{{% tab "Docker" %}}
-
-Add the following configuration block in the `datadog.yaml` [configuration file][1].
+The Agent automatically detects reachable sockets and API Endpoints (Docker, containerd, Kubernetes API, etc.) and activates Autodiscovery for you.
+If Autodiscovery is not working, you can verify the detected features by running `agent status`.
 
+In case the automatic detection failed or if you want to deactivate automatically detected features, you can use these configuration parameters in `datadog.yaml` to include/exclude features:
```

Comment:
```suggestion
In case the automatic detection failed or you want to deactivate automatically detected features, use these configuration parameters in `datadog.yaml` to include/exclude features:
```